src/reporting-flow/sub-components/next-steps-step.tsx

```diff
@@ -12,19 +15,40 @@ interface Props {
 export function NextStepsStep( { stepNumber }: Props ) {
 	const relevantTaskIds = useAppSelector( selectRelevantTaskIds );
 	const allTasksAreComplete = useAppSelector( selectAllTasksAreComplete );
+	const issueFeatureId = useAppSelector( selectIssueFeatureId );
+	const issueType = useAppSelector( selectIssueType );
+	const activeStep = useAppSelector( selectActiveStep );
 
 	// It's a no-op since we don't use the edit button in this step!
 	const onEdit = useCallback( () => {
 		return;
 	}, [] );
 
 	const tasksExist = relevantTaskIds.length > 0;
+	const requiredInfoIsMissing =
+		( issueFeatureId === null || issueType === 'unset' ) && activeStep === 'nextSteps';
+	const noTasksAreConfigured = ! tasksExist && activeStep === 'nextSteps';
 
 	const isComplete = tasksExist && allTasksAreComplete;
 
 	let stepContentDisplay: ReactNode;
 	if ( tasksExist ) {
 		stepContentDisplay = <NextSteps />;
+	} else if ( requiredInfoIsMissing ) {
+		stepContentDisplay = (
+			<p className={ styles.badStateMessage } role="alert">
+				Hmm... We seem to be missing some information. This almost always means we could not parse
+				the info in the URL. Restart by clicking on the Bugomattic header at the top of the page.
```

Comment:
**suggestion (non-blocking):** Disclaimer: I'm not an expert on copy haha. I'm wondering if we could update the last sentence to make it clearer what the next steps are: 1) recheck the URL for typos or 2) start over.

> "Hmm, we seem to be missing some information. This almost always means we could not parse the info in the URL. Please check the URL for typos or start over using the Bugomattic header at the top of the page."

...or something like that 😅

Reply:
Sounds good! I don't think we'll want to guide them to fix the URL though, because that will be a losing battle -- the way the state is serialized can be really hard to reproduce by hand! But I'll be more explicit about starting over...

How about:

> Hmm, we seem to be missing some information. This almost always means we could not parse the info in the URL. Please start over by clicking the Bugomattic header at the top of the page.

Reply:
> I don't think we'll want to guide them to fix the URL though, because that will be a losing battle

Ah, that makes sense to me. I think your suggested copy sounds great!